actions/registerRealm.ts

```diff
@@ -97,15 +128,27 @@ async function prepareMintInstructions(
     // then should create mints to them
     if (otherOwners?.length) {
       for (const ownerPk of otherOwners) {
-        const ataPk = await withCreateAssociatedTokenAccount(
-          mintInstructions,
-          _mintPk,
+        const ata: ProgramAccount<AccountInfo> | undefined = await tryGetAta(
+          connection,
           ownerPk,
-          walletPubkey
+          _mintPk
         )
+        const shouldMint = !ata?.account.amount.gt(new BN(0))
+
+        const ataPk =
+          ata?.publicKey ??
+          (await withCreateAssociatedTokenAccount(
```

Comment:
If the Ata was created before the this instruction would fail, you have to check for Ata existence separately, and if it exists then skip this instruction